bigbluebutton-html5/client/main.jsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
+import htmlBBB from '/imports/api/bbb';
```

Comment:
The bbb api is a javascript api not a react component. I believe it being imported here executes the final line which initialises it. Its a plain javascript file so unless its imported into the project it doesnt execute.

I guess the class could be imported, then the initialisation done in the place of import